docker-multinode/common.sh

```diff
@@ -45,7 +45,7 @@ kube::multinode::main(){
     -v /var/run:/var/run:rw \
     -v /:/rootfs:ro \
```

Comment:
Any chance you may test if it affects the results? I think it's there just for `--containerized`